src/APIResponse.php

```diff
@@ -21,14 +55,16 @@ public function __construct()
     }
 
     /**
-     * @param $status
-     * @param $message
-     * @param $data
-     * @param array $extraData
+     * Create API response.
+     *
+     * @param int    $status
+     * @param string $message
+     * @param array  $data
+     * @param array  $extraData
      *
      * @return JsonResponse
      */
-    public function response($status, $message, $data, ...$extraData)
+    public function response($status = 200, $message = null, array $data = [], ...$extraData)
```

Comment:
@obiefy  I think by changing this to default to an array for $data has introduced a breaking change.

I now get this error
```
\TypeError: Argument 3 passed to Obiefy/API/APIResponse::response() must be of the type array, object given, called in /Users/beauraines/projects/dog-walker/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 261
```

when using the example from https://github.com/obiefy/api-response#advanced-usage in `artisan tinker`

I got the same thing in my project and was able to resolve it by casting my object to an array with `toArray()`. I'm not sure if this was the intended case, it worked before with either an Object or an Array.

Reply:
@beauraines you are totally right. this will introduce a breaking change.
let me remove it.

Reply:
@beauraines Kindly check it now, and let me know.
And please if you have any comments for this PR, let me know so we can improve it together.
Thanks.

Reply:
Yep! That fixed it.

On my fork of this branch, I wrote a test for this as well. I'll submit a PR.